core/charm/computedseries.go

```diff
@@ -20,13 +20,6 @@ func ComputedSeries(c charm.CharmMeta) ([]string, error) {
 		return c.Meta().Series, nil
 	}
 
-	// If we have V2 metadata *and* a non-empty containers collection,
-	// then this is a side-car based charm and we return "kubernetes"
-	// instead of translating the collection of supplied bases.
-	if IsKubernetes(c) {
-		return []string{coreseries.Kubernetes.String()}, nil
-	}
```

Comment:
What was the reason for this removal?

Reply:
it's wrong to force "kubernetes" as the series for a v2 k8s charm. The series is used to form the tag of the charm base image to download, which needs to be "ubunut-20.04", not "kubernetes-kubernetes"

Reply:
Ace, thought as much after to speaking to @manadart